packages/nimble-components/tsconfig.json

```diff
@@ -20,6 +20,7 @@
         // so imports with side-effects are explicit
         "importsNotUsedAsValues": "error",
         "noImplicitOverride": true,
+        "skipLibCheck": true,
```

Comment:
Discussed with @rajsite, and as expected we wouldn't be able to submit this. Some possibilities for moving forward include:
- Have an acceptable alternative to using arquero that meets all of the necessary performance requirements
- Providing our own typing for arquero that addresses the known typescript errors.
- Addressing the typescript issues in arquero itself by submitting a PR in arquero's repo.

@rajsite, are there are any other options worth considering?

Reply:
I implemented an alternative for index searching.
I also created a PR with some type fixes in arquero. https://github.com/uwdata/arquero/pull/346
If you have time for some insights over there please let me know

Reply:
Could you leave a comment in the code where you intend to ultimately switch to using arquero? Ultimately, I would expect to have an issue created in Nimble that is blocked on the PR you submitted to arquero, which we can then reference in the TODO comment.